npm2deb/__init__.py

```diff
@@ -107,6 +107,9 @@ def initiate_build(self, saved_path):
             self.edit_changelog()
             self.create_git_repository()
 
+            if not _os.path.exists(_os.path.join(saved_path, self.name, new_dir)):
```

Comment:
A better approach would be to make `create_git_repository` return a flag indicating the success and use that to update `new_dir`

something like 
```
status = self.create_git_repository()
if status:
    new_dir = self.debian_name
```

Reply:
@shanavas786 thanks for the feedback, I can do so if you want to. However if you notice, all the functions in `__init__.py` are mostly used for invoking some function, and _ideally_ at least some of them should return some or the other flags indicating success/failure if we go by this logic. None of the functions have a return value as such, and are intended to just run some functionality.

Adding such a flag to this function makes the code look a little ugly (to my taste) and the above `if` condition IMHO is sufficient to indicate that a git repo hasn't been created. If you still think I should add it, please tell me and I'll push a commit. Otherwise please consider merging.